runtime/parachains/src/paras.rs

```diff
@@ -412,15 +412,28 @@ impl<T: Trait> Module<T> {
 	/// Schedule a para to be cleaned up at the start of the next session.
 	#[allow(unused)]
 	pub(crate) fn schedule_para_cleanup(id: ParaId) -> Weight {
-		OutgoingParas::mutate(|v| {
+		let upcoming_weight = UpcomingParas::mutate(|v| {
+			match v.binary_search(&id) {
+				Ok(i) => {
+					v.remove(i);
```

Comment:
Should remove from `UpcomingParasGenesis` as well, right?

Reply:
true, thanks